src/toolbar_gui.cpp

```diff
@@ -552,9 +552,7 @@ static CallBackFunction ToolbarSubsidiesClick(Window *w)
  */
 static CallBackFunction MenuClickSubsidies(int index)
 {
-	switch (index) {
-		case 0: ShowSubsidiesList(); break;
```

Comment:
Looks like this one was not really working outside of single player mode (hardcoded to company 0 only)

Reply:
I believe this is dropdown index 0? (Subsidy button has only one dropdown)

Reply:
Yes, there is only one button in the dropdown, as it's a list of subsidies for everyone. It doesn't care about the compan when opening it.